src/views/Housing/components/RAView/components/Schedule/index.jsx

```diff
@@ -1,7 +1,53 @@
 import ScheduleCalendar from '/src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx';
+import { Calendar, luxonLocalizer } from 'react-big-calendar';
+import { DateTime } from 'luxon';
```

Comment:
We should be using built in functions, 360 is trying to phase out Luxon (According to comment from Evan)

Reply:
Removed `Schedule` for now, since it's not going to be in the MVP, I'll figure out a replacement for Luxon next semester.